### PR TITLE
Clarify FUTURE crypto-policy

### PR DESCRIPTION
**Pull request by ehelms**
* [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/common/modules/ref_system-requirements.adoc [hunk 1]
```diff
@@ -119,7 +119,7 @@ For more information, see https://access.redhat.com/documentation/en-us/red_hat_
 [NOTE]
 ====
 {Project} supports DEFAULT and FIPS crypto-policies.
-The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations.
+The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations. The FUTURE policy is a stricter forward-looking security level intended for testing a possible future policy. For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening[Chapter 4. Using system-wide cryptographic policies].
```
Comment (mjivraja) [inline guides/common/modules/ref_system-requirements.adoc:122]:
```suggestion
The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations. 
The FUTURE policy is a stricter forward-looking security level intended for testing a possible future policy. 
For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening[Using system-wide cryptographic policies] in {EL} guide.
```

**Review by mjivraja — COMMENTED** [review]
A minor change.

File: guides/common/modules/ref_system-requirements.adoc (hunk 1 above)
Comment (ehelms) [reply guides/common/modules/ref_system-requirements.adoc]: Thanks, I've updated the PR.

**Review by adamlazik1 — APPROVED** [review]

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   b68b176e3..2f3c924dd  3.7 -> 3.7

**Pull request merged**